xmrstak/backend/nvidia/nvcc_code/cuda_extra.cu

```diff
@@ -556,14 +564,14 @@ extern "C" int cuda_get_deviceinfo(nvid_ctx* ctx)
 		err = cudaSetDevice(ctx->device_id);
 		if(err != cudaSuccess)
 		{
-			printf("WARNING: NVIDIA GPU %d: cannot be selected.\n", ctx->device_id);
+			printf("SKIP! (cudaSetDevice failed)\n");
 			return 2;
 		}
 		// trigger that a context on the gpu will be allocated
 		err = cudaMalloc(&tmp, 256);
 		if(err != cudaSuccess)
 		{
-			printf("WARNING: NVIDIA GPU %d: context cannot be created.\n", ctx->device_id);
+			printf("SKIP! (cudaMalloc failed)\n");
```

Comment:
please revert this, also the WARNING is important because the word warning is somewhere defined in the docs.

Reply:
But it literally skips the GPU then?
Warnings are for a case where it can continue but degraded (no-AES)
You can't warning while also taking the failure action (no GPU init).